tests/h/views/api/groups_test.py

```diff
@@ -260,38 +241,35 @@ def pyramid_request(self, pyramid_request, factories):
     def group(self, factories):
         return factories.Group(authority="example.com")
 
+    @pytest.fixture
```

Comment:
I think scoping the fixture closer to the tests it's used in reduces the bad side of this equation too.

I'll check, because I wouldn't be sure it's a guarantee, that the tests work if we turn it on all the time. Some of the `h` tests are pretty weird.

Reply:
Mildly interesting. This was at the bottom of the file and was `usefixture`'d in these tests. So I moved it locally because they are the only ones that use it and missed the use fixture mark. So actually this was `autouse` all along effectively.